easybuild/easyconfigs/t/Tk/Tk-8.6.4-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
```

Comment:
As discussed, this patch fixes Tk and here the broken thing seems to be the search on setupext.py from matplotlib.